docs/05-technical-reference/03-architecture/obsv-logging-architecture.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I just realized that the "tails logs" arrow is in the wrong direction, as it is a "reading of data" (not a HTTP-request)